lib/util/exec/index.ts

```diff
@@ -86,9 +86,10 @@ export async function exec(
 
   // TODO: simplify after callers' refactoring
   const singleCommand = typeof cmd === 'string' ? cmd : cmd.join(' && ');
-  let pExecCommand = singleCommand.startsWith('docker run')
-    ? singleCommand
-    : `bash -l -c "${singleCommand.replace(/"/g, '\\"')}"`;
+  let pExecCommand =
+    singleCommand.startsWith('docker run') || process.platform === 'win32'
```

Comment:
I'm running on windows and without this it would attempt to run bash on windows which via cmd fails

Reply:
In the latest master it should not use bash for local commands